README.md

```diff
@@ -336,6 +335,23 @@ allow operation with the unoptimized Javascript files.
 Debugging can be enabled by creating the file
 `specify7/specifyweb/settings/debug.py` with the contents, `DEBUG = True`.
 
+**Backend:**
+Enabling `SP7_DEBUG` enables DEBUG mode for Django
+<https://docs.djangoproject.com/en/3.2/ref/settings/#debug>
+
+The Specify backend also special behavior while in DEBUG mode:
+- When the backend encounters an error, Django returns an error message containing comprehensive information of the error and potentially sensitive system information
+- Specify validates the WorkBench upload plan when fetching the WorkBench's upload results
+   - From the `/api/workbench/upload_results/(?P&lt;ds_id&gt;\d+)/` endpoint
+- Specify adds an indent to the datamodel returned from the `context/datamodel.json` endpoint
+- (DISABLED BY DEFAULT) Specify allows any endpoint/view to be profiled by adding a `/?prof` query parameter
```

Comment:
I am not sure how thorough you wish to be, but just having it adds an indent and formats the datamodel returned from the endpoint might just be taking up space. 

Someone from @specify/dev-testing or @specify/developers will have to take a look at the profiling middleware and see if it's even functional when enabled. 
Theoretically, it just needs to be enabled by removing the comment in `specifyweb/settings/__init__.py`
https://github.com/specify/specify7/blob/52d0f1c12b294936382ef90cc24ec825ce0a6e9d/specifyweb/settings/__init__.py#L190